applications/CableNetApplication/python_scripts/edge_cable_element_process.py

```diff
@@ -0,0 +1,40 @@
+import KratosMultiphysics as KratosMultiphysics
+import KratosMultiphysics.CableNetApplication as CableNetApplication
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return EdgeCableElementProcess(Model, settings["Parameters"])
+
+
+
+class EdgeCableElementProcess(KratosMultiphysics.Process):
+
+    def __init__(self, Model, settings ):
+        KratosMultiphysics.Process.__init__(self)
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "model_part_name"           : "example_part",
+            "computing_model_part_name" : "computing_domain",
+            "element_type"              : "cable",
+            "node_id_order"             : [1,2,3],
```

Comment:
I have no idea what you are doing but this seems difficult to use

Reply:
you need to give it the correct node order, as this is an element with arbitrary nodes